### ISSUE TITLE
[DEVELOP] Add a provider for keeping different settings persistent

### ISSUE THREAD

**Issue by nathanlesage** [Zettlr/Zettlr#1039]
A lot of people complain about Zettlr not really remembering a lot of positions such as the position of the sidebar divider, the window position, yadda yadda.

Currently, some metrics (window positions) are managed by the config provider but that's bad, so we should outsource this feature to a different provider.

Not complicated, but it will make our lives much easier in the future!

**Comment by BeatLink** [comment]
Any updates on this?

**Comment by nathanlesage** [comment]
Well, in 1.8 I shipped a feature that would persist the window positions, which was the first thing. The rest'll follow slowly within the coming months.

**Comment by anaelhamsa** [comment]
Hey ! First of all, thank you so much for Zettlr! I just started my PhD and it already is my daily driver to take notes, save R codes and write parts of my manuscript. That's awesome. 
(I'm really new to this GitHub thingy and just starting to learn how to act in this ecosystem. Bear with me here...)

I think I'm in the right place and that the issue has already been reported, but let me quickly describe what I found on my part. 
My problem is that Zettlr saves the window position from the last working session, and it stays "out of reach" when I go from a multi screen to a simple screen display (laptop). To get the Zettlr window back, I have to plug my laptop back to the second screen and drag it back to the main screen. 
(Might be the same case treated in #1322 (closed).
Thanks by advance. Cheers.
